tensorrt_llm/_torch/auto_deploy/shim/ad_executor.py

```
@@ -324,11 +323,17 @@ def forward(
-            [ls_one_seq[-last_only:] for ls_one_seq, last_only in zip(logits, last_logit_only)],
-            dim=0,
-        )
+        if self.cache_seq_interface.info.is_generate:
+            logits = logits.squeeze(1)  # [b,1,vocab_size]->[b,vocab_size]
+        else:
+            # mixed prefill and decode
+            logits = logits.squeeze(0)
+            logits = list(torch.split(logits, self.cache_seq_interface.info.seq_len))
+            # gather+cat logits
+            logits_flat = torch.cat(
+                [ls_one_seq[-last_only:] for ls_one_seq, last_only in zip(logits, last_logit_only)],
```

Ruff (0.14.7)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)